ruby/private/bundle/create_bundle_build_file.rb

```diff
@@ -78,18 +64,19 @@ def create_bundle_build_file(build_out_file, lock_file, repo_name, excludes, wor
   bundle.specs.each { |spec|
     deps = spec.dependencies.map(&:name)
     deps += [":bundler_setup"]
-    template_out += GEM_TEMPLATE.gsub("{exclude}", excludes)
+
+    exclude_array = excludes[spec.name] || []
+    # We want to exclude files and folder with spaces in them
+    exclude_array += ["**/* *.*", "**/* */*"]
```

Comment:
Why can't we work with folder with spaces in them? Just curious.

Reply:
Bazel does not like them because it creates a runfile target name out of the file path which breaks it.